arcgis-ios-sdk-samples/Geometry/Nearest vertex/NearestVertexViewController.swift

```diff
@@ -95,52 +99,74 @@ class NearestVertexViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        addGraphicsToOverlay()
         // Add the source code button item to the right of navigation bar.
-        (self.navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = ["NearestVertexViewController"]
+        (navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = ["NearestVertexViewController"]
+        addGraphicsToOverlay()
```

Comment:
This is a very odd place for this. Instead of a method that adds graphics to the existing overlay, I would suggest a method that _creates_ the graphics overlay--that way it doesn't need to be an instance property:

```swift
func makeGraphicsOverlay() -> AGSGraphicsOverlay {
    let polygonFillSymbol = AGSSimpleFillSymbol(
        style: .forwardDiagonal,
        color: .green,
        outline: AGSSimpleLineSymbol(style: .solid, color: .green, width: 2)
    )
    let polygonGraphic = AGSGraphic(geometry: polygon, symbol: polygonFillSymbol)
    
    let graphicsOverlay = AGSGraphicsOverlay()
    graphicsOverlay.graphics.addObjects(from: [
        polygonGraphic,
        nearestCoordinateGraphic,
        tappedLocationGraphic,
        nearestVertexGraphic
    ])
    return graphicsOverlay
}
```

And then add it to the map like so:

```swift
mapView.graphicsOverlays.add(makeGraphicsOverlay())
```